performance_manager/lib/gtfs_static_table.py

```diff
@@ -0,0 +1,317 @@
+    columns_to_pull = [
+        "stop_id",
+        "stop_name",
+        "stop_desc",
+        "platform_code",
+        "platform_name",
+        "parent_station",
+        "timestamp",
+    ]
+    int64_cols = [
+        "timestamp",
+    ]
+    get_and_insert_static_tables(
+        table_name,
+        feed_info_path,
+        session,
+        insert_table,
+        columns_to_pull=columns_to_pull,
+        int64_cols=int64_cols,
+    )
+
+
+def process_stop_times(
+    feed_info_path: str, session: sa.orm.session.sessionmaker
+) -> None:
+    """
+    load stop_times table
+    """
+    table_name = "STOP_TIMES"
+    insert_table = StaticStopTimes.__table__
+    columns_to_pull = [
+        "trip_id",
+        "arrival_time",
+        "departure_time",
+        "stop_id",
+        "stop_sequence",
+        "timestamp",
+    ]
+    int64_cols = [
+        "stop_sequence",
+        "timestamp",
+    ]
+    time_to_seconds_cols = [
+        "arrival_time",
+        "departure_time",
+    ]
+    get_and_insert_static_tables(
+        table_name,
+        feed_info_path,
+        session,
+        insert_table,
+        columns_to_pull=columns_to_pull,
+        int64_cols=int64_cols,
+        time_to_seconds_cols=time_to_seconds_cols,
+    )
+
+
+def process_calendar(
+    feed_info_path: str, session: sa.orm.session.sessionmaker
+) -> None:
+    """
+    load calendar table
+    """
+    table_name = "CALENDAR"
+    insert_table = StaticCalendar.__table__
+    columns_to_pull = [
+        "service_id",
+        "monday",
+        "tuesday",
+        "wednesday",
+        "thursday",
+        "friday",
+        "saturday",
+        "sunday",
+        "start_date",
+        "end_date",
+        "timestamp",
+    ]
+    int64_cols = [
+        "start_date",
+        "end_date",
+        "timestamp",
+    ]
+    bool_cols = [
+        "monday",
+        "tuesday",
+        "wednesday",
+        "thursday",
+        "friday",
+        "saturday",
+        "sunday",
+    ]
+    get_and_insert_static_tables(
+        table_name,
+        feed_info_path,
+        session,
+        insert_table,
+        columns_to_pull=columns_to_pull,
+        int64_cols=int64_cols,
+        bool_cols=bool_cols,
+    )
+
+
+def process_static_tables(sql_session: sessionmaker) -> None:
+    """
+    process gtfs static table files from metadataLog table
+    """
+
+    # pull list of objects that need processing from metadata table
+    paths_to_load = get_unprocessed_files("FEED_INFO", sql_session)
+
+    for folder, folder_data in paths_to_load.items():
+        ids = folder_data["ids"]
+        folder = str(folder)
+
+        try:
+            process_feed_info(folder, sql_session)
+            process_trips(folder, sql_session)
+            process_routes(folder, sql_session)
+            process_stops(folder, sql_session)
+            process_stop_times(folder, sql_session)
+            process_calendar(folder, sql_session)
```

Comment:
process each static schedule table

Reply:
i think it might be better to do the E and T before the L here?

in the event that any of the later `process_<table_name>` functions throws, do we want data to be written to the tables from the earlier steps?

Reply:
breaking the E, T, and L out would also make each step more testable.